### PR TITLE
chore: configure release-it

### PR DESCRIPTION
**Pull request by troZee**
# Summary

Add `release-it` to automate release process. 

## Test Plan

run `yarn release-it --dry-run` in project dir

### CODE REVIEW

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -210,6 +210,18 @@ dependencies {
         implementation jscFlavor
     }
 }
+task downloadDependencies() {
+    description 'Download all dependencies to the Gradle cache'
+    doLast {
+        configurations.findAll().each { config ->
+            if (config.name.contains("minReactNative") && config.canBeResolved) {
+                print config.name
```
Comment (Krizzu) [inline example/android/app/build.gradle:218]: do you need `print` here?

File: package.json [hunk 2]
```diff
@@ -11,7 +11,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "flow": "flow",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix --ext .js"
+    "lint:fix": "eslint . --fix --ext .js",
```
Comment (Krizzu) [inline package.json:14]: I think `.js` is default, so no need to specify

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,183 +1,60 @@
...
+      - rn/android_build:
+          name: build_android
+          project_path: "example/android"
+          build_type: debug
           requires:
-            - linux-checkout
-      # - jest:
+            - analyse_js
+
+      # Build the iOS app in release mode and do not run tests
```
Comment (wojteg1337) [inline .circleci/config.yml:51]: Do you need it?

Comment (troZee) [reply .circleci/config.yml:51]: I would like to check, if iOS building correctly, but probably, this repo has no permission to macOS container. Now I have no time to check it. I would like to do it later.

**Pull request merged**